jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -110,15 +120,45 @@ def _set_default_results_label(self, event={}):
         label_comps += [f"moment {self.n_moment}"]
         self.results_label_default = " ".join(label_comps)
 
-    @observe("dataset_selected")
+    @observe("dataset_selected", "n_moment")
     def _set_data_units(self, event={}):
+
```

Comment:
When using the UI, if you delete the number in the moment field, you get the following traceback:
```
File ~/git/jdaviz/jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py:126, in MomentMap._set_data_units(self, event)
    123 @observe("dataset_selected", "n_moment")
    124 def _set_data_units(self, event={}):
--> 126     unit_options_index = 2 if self.n_moment > 2 else self.n_moment
    127     if self.output_unit_selected not in moment_unit_options[unit_options_index]:
    128         self.output_unit_selected = moment_unit_options[unit_options_index][0]

TypeError: '>' not supported between instances of 'str' and 'int'
```
because `self.n_moment` is an empty string. Here's a possible fix?


```suggestion
        if isinstance(self.n_moment, str): 
            return
```